OpenDreamClient/Interface/InterfaceMenu.cs

```diff
@@ -106,16 +116,53 @@ public sealed class MenuElement : InterfaceElement {
             if (String.IsNullOrEmpty(text))
                 return new MenuBar.MenuSeparator();
 
+            if(MenuElementDescriptor.CanCheck)
+                if(MenuElementDescriptor.IsChecked)
+                    text =  text + " ☑";
+
             MenuBar.MenuButton menuButton = new() {
                 Text = text
             };
 
-            //result.IsCheckable = MenuElementDescriptor.CanCheck;
-            if (!String.IsNullOrEmpty(Command))
-                menuButton.OnPressed += () => { EntitySystem.Get<DreamCommandSystem>().RunCommand(Command); };
+
+            menuButton.OnPressed += () => {
+                    if(MenuElementDescriptor.CanCheck)
+                        if(MenuElementDescriptor.Group != null)
```

Comment:
According to documentation, a blank group is also treated as no group.
```suggestion
                        if(!String.IsNullOrEmpty(MenuElementDescriptor.Group))
```